examples/notebooks/GalSimTutorialCustomCatalog.ipynb

```diff
@@ -367,7 +367,7 @@
     "import time\n",
     "from lsst.sims.utils import ObservationMetaData\n",
     "start = time.clock()\n",
-    "obs = ObservationMetaData(unrefractedRA=52.0, unrefractedDec=-26.0,\n",
+    "obs = ObservationMetaData(pointingRA=52.0, pointingDec=-26.0,\n",
```

Comment:
Hey, this is a notebook and github didn't show a ton of junk! I wonder if they got better at making notebook diffs or if this was just a simple one that didn't change much.